ludwig/utils/tokenizers.py

```diff
@@ -50,14 +46,61 @@ def __call__(self, text: str):
         pass
 
 
-class CharactersToListTokenizer(BaseTokenizer):
-    def __call__(self, text):
-        return [char for char in text]
+class StringSplitTokenizer(torch.nn.Module):
+    def __init__(self, split_string, **kwargs):
+        super().__init__()
+        self.split_string = split_string
+
+    def forward(self, v: Union[str, List[str], torch.Tensor]) -> Any:
+        if isinstance(v, torch.Tensor):
+            raise ValueError(f"Unsupported input: {v}")
+
+        inputs: List[str] = []
```

Comment:
I see that you are adapting an existing implementation, though this seems more complicated than I would expect (for example, why do we have a `get_tokens()` function that returns its own input?). 

@geoffreyangus, ooc does this also look strange to you, or is this imposed on us by torchscript?

Reply:
It looks like `NgramTokenizer`, which subclasses `SpaceStringToListTokenizer` (which in turn subclasses the new `StringSplitTokenizer`, seems to override `get_tokens`: https://github.com/ludwig-ai/ludwig/pull/3602/files#diff-5cbace55f4f4fd07725c061b9f981b83fe43cb53b0045cf1257c9fb5d4931f0dR132-R142